features/quest/components/QuestTypeModal.tsx

```diff
@@ -69,13 +81,13 @@ const QuestTypeModal = ({
           {!questModalOpen && (
             <>
               <SelectField
-                label="Sichbarkeit"
+                label="Sichtbarkeit"
```

Comment:
Revert this change since this is changed with i18n
```suggestion
                label="Sichbarkeit"
```